README.md

```diff
@@ -14,6 +14,32 @@ vary across manufacturers, devices and android versions. This library was create
 to provide a consistent experience across all devices and users.
 
 
+## Installation
+
+Step 1. Add the JitPack repository to your build file
+
+```groovy
+allprojects {
+  repositories {
+    ...
+    maven { url 'https://jitpack.io' }
+  }
+}
+```
+
+Step 2. Add the dependency
+
+```groovy
+dependencies {
+    implementation 'com.github.prolificinteractive.geocoder:geocoder:0.1.0'
```

Comment:
why do we have 3 dependencies that are not optional? I feel like one dependency should be not optional and then you have a bunch of optional.